src/main/java/com/courseApp/reviewService/ReviewRequestProcessor.java

```diff
@@ -23,7 +29,25 @@
      */
     @Override
     public Map<String, String> queryCourseReviewSummary(String courseCode) {
-        return null;
+        ReviewDAO reviewDao = new ReviewDaoImpl();
+        CourseReview courseReview = reviewDao.queryCourseReview(courseCode);
+        if (courseReview != null){
+
+            // Create map to store summary values
+            Map<String, String> summary = new HashMap<>();
+
+            // Add keys and values to the summary map
+            summary.put("courseCode", courseCode);
```

Comment:
Code looks good but may move the constants to `Constants.class` under our constant folder.